Tools/CISupport/ews-build/steps.py

```diff
@@ -54,9 +54,9 @@
 RESULTS_DB_URL = 'https://results.webkit.org/'
 WithProperties = properties.WithProperties
 Interpolate = properties.Interpolate
-BRANCH_PR_RE = re.compile(r'^refs/pull/(?P<id>\d+)/merge$')
 GITHUB_URL = 'https://github.com/'
 GITHUB_PROJECTS = ['WebKit/WebKit']
+HASH_SIZE = 12
```

Comment:
maybe HASH_SIZE  => HASH_LENGTH_TO_DISPLAY.
Also probably 8 is fine to display in buildbot UI, from readability perspective (even though there is very slight chance of collision with 8 char len)